common/components/TXMetaDataPanel/components/AdvancedGas.tsx

```diff
@@ -107,18 +119,58 @@ class AdvancedGas extends React.Component<Props, State> {
           </div>
         )}
 
-        {feeSummary && (
-          <div className="AdvancedGas-fee-summary">
-            <FeeSummary
-              gasPrice={gasPrice}
-              render={({ gasPriceWei, gasLimit, fee, usd }) => (
+        {this.renderFee()}
+      </div>
+    );
+  }
+
+  private renderFee() {
+    const { gasPrice, scheduling, scheduleGasPrice, timeBounty } = this.props;
+    const { feeSummary } = this.state.options;
+
+    if (!feeSummary) {
+      return;
+    }
+
+    if (scheduling) {
+      return (
+        <div className="AdvancedGas-fee-summary">
+          <SchedulingFeeSummary
+            gasPrice={gasPrice}
+            scheduleGasPrice={scheduleGasPrice}
+            render={({ gasPriceWei, scheduleGasLimit, fee, usd }) => (
+              <div>
                 <span>
-                  {gasPriceWei} * {gasLimit} = {fee} {usd && <span>~= ${usd} USD</span>}
+                  <UnitDisplay
+                    value={EAC_SCHEDULING_CONFIG.FEE.mul(EAC_SCHEDULING_CONFIG.FEE_MULTIPLIER)}
+                    unit={'ether'}
+                    displayShortBalance={true}
+                    checkOffline={true}
+                    symbol="ETH"
+                  />{' '}
+                  + {timeBounty && timeBounty.value && timeBounty.value.toString()} + {gasPriceWei}{' '}
+                  * {EAC_SCHEDULING_CONFIG.SCHEDULING_GAS_LIMIT.toString()} +{' '}
+                  {scheduleGasPrice && scheduleGasPrice.value && scheduleGasPrice.value.toString()}{' '}
+                  * ({EAC_SCHEDULING_CONFIG.FUTURE_EXECUTION_COST.toString()} + {scheduleGasLimit})
+                  =&nbsp;{fee}&nbsp;{usd && <span>~=&nbsp;${usd}&nbsp;USD</span>}
                 </span>
-              )}
-            />
-          </div>
-        )}
+              </div>
+            )}
+          />
+        </div>
+      );
+    }
+
+    return (
+      <div className="AdvancedGas-fee-summary">
+        <FeeSummary
+          gasPrice={gasPrice}
+          render={({ gasPriceWei, gasLimit, fee, usd }) => (
+            <span>
+              {gasPriceWei} * {gasLimit} = {fee} {usd && <span>~= ${usd} USD</span>}
```

Comment:
Remove `$`

Reply:
Nope, it's meant to be like that. Please see the current version, it displays $ in front of value and USD after the value.

Clarification: by current version I mean develop or beta.mycrypto.com (v0.7.0 beta).